components/companies-using-kedro/companies-using-kedro.module.scss

```diff
@@ -17,100 +17,54 @@
 
 .inner {
   background: $color-dark;
-  font-size: 3rem;
+  flex-direction: column;
+  font-size: 4.5rem;
   font-weight: 700;
   line-height: 6rem;
   display: flex;
   margin: 0 auto;
   max-width: $max-content;
 
-  @include breakpoint(medium-only) {
-    flex-direction: column;
-  }
-
   @include breakpoint(small) {
-    flex-direction: column;
     font-size: 1.5rem;
     line-height: 3rem;
   }
 }
 
-.left {
-  align-items: center;
-  justify-content: right;
-  display: flex;
-  padding-right: 1.5rem;
-  width: 45%;
-
-  @include breakpoint(medium-only) {
-    justify-content: left;
-    width: 100%;
-  }
-
-  @include breakpoint(small) {
-    justify-content: left;
-    width: 100%;
-  }
-}
-
-.right {
-  overflow: hidden;
-  position: relative;
-  width: 50%;
-
-  @include breakpoint(medium-only) {
-    width: 100%;
-  }
-
-  @include breakpoint(small) {
-    width: 100%;
-  }
-}
-
-.animation {
-  position: absolute;
-  top: 40%;
-}
-
-.animationMask {
-  background-image: linear-gradient(
-    $color-dark,
-    transparent,
-    transparent,
-    $color-dark
-  );
-  height: 30rem;
-  pointer-events: none;
-  position: relative;
+.title {
+  margin-bottom: 3rem;
+  text-align: center;
```

Comment:
We have a mixin that you can use here to make life easier. The font is now too small on mobile compared to the other section headers. [Reference link here](https://github.com/kedro-org/kedro-website/blob/main/components/features/features.module.scss#L24).

Reply:
ah great thanks for letting me know 👍